### PR TITLE
Актуализация доки по бэку

### PR DESCRIPTION
**Pull request by grigoriev-semyon**

### CODE REVIEW

File: docs/tvoy-ff/backend/index.md [hunk 1]
```diff
@@ -3,108 +3,246 @@
...
+Для работы с БД мы используем `SQLAlchemy`. Это очень мощный инструмент разработки с встроенным ORM. Смысл ORM - установка соответствий между моделями с БД и объектами в python. Таким образом, вы работаете с экземплярами соответствующих классов, а не с чистыми SQL-запросами, которые зачастую очень громоздкие и неудобные в дальнейшей работе с ними.
 
-Для начала, создаются все модели(классы) в python. Путь к ним обычно выглядит как /models/db.py. Поля таких классов - колонки. При создании таблиц `sqlalchemy.Column` создает нужную колонку, а в дальнейшем при обращении к этому полю, выдает значение для текущей строки в БД.
+Для начала, создаются все модели(классы) в python. Путь к ним обычно выглядит как /models/db.py. Поля таких классов - колонки. При создании таблиц `sqlalchemy.mapped_column` создает нужную колонку, а в дальнейшем при обращении к этому полю, выдает значение для текущей строки в БД.
 https://docs.sqlalchemy.org/en/20/orm/quickstart.html#declare-models
```
Comment (dyakovri) [inline docs/tvoy-ff/backend/index.md:48]: <>

File: docs/tvoy-ff/backend/index.md [hunk 2]
```diff
@@ -3,108 +3,246 @@
...
 
-После создания моделей, описываются relationships между ними. Это нужно для простой и быстрой реализации больших и, кажется, ненужных запросов. То есть, `childrens = session.query(Parent).filter(Children.parent_id == parent.id).all()` превращается в `parent.childrens`. Очень упрощает читаемость кода.
+После создания моделей, описываются `relationships` между ними. Это нужно для простой и быстрой реализации больших и, кажется, ненужных запросов. То есть, `childrens = session.query(Parent).filter(Children.parent_id == parent.id).all()` превращается в `parent.childrens`. Очень упрощает читаемость кода.
 https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html
```
Comment (dyakovri) [inline docs/tvoy-ff/backend/index.md:51]: <>

File: docs/tvoy-ff/backend/index.md [hunk 3]
```diff
@@ -3,108 +3,246 @@
...
+    ```bash
+    alembic upgrade head
+    ```
 
 ## Как работаем с вебом
 
-Для работы с вебом мы используем __FastAPI__. В нем есть все необходимое и весит он меньше, чем Django :)
+Для работы с вебом мы используем __FastAPI__.
 
 Работаем мы по стандартам REST API: https://habr.com/ru/post/483202/
```
Comment (dyakovri) [inline docs/tvoy-ff/backend/index.md:79]: <>

**Comment by grigoriev-semyon** [comment]
related: https://github.com/profcomff/.github/issues/114

**Review by dyakovri — APPROVED** [review]

File: docs/tvoy-ff/backend/config.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class Settings(BaseSettings):
+    """Application settings"""
+
+    DB_DSN: PostgresDsn = 'postgresql://postgres@localhost:5432/postgres'
+    model_config = ConfigDict(case_sensitive=True, env_file=".env", extra="allow")
+```
+
+Пример .env файла, например, из бэкенда расписания: https://github.com/profcomff/timetable-backend/blob/main/.env.example.
```
Comment (dyakovri) [inline docs/tvoy-ff/backend/config.md:24]: <>

File: docs/tvoy-ff/backend/config.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Как работаем с конфигом
+
+Для конфигурации мы используем переменные окружения. Чтобы многократно не заполнять них, мы используем файлы окружения. Для работы с файлами окружения мы используем ``pydantic-settings``. В файле settings.py создается класс Settings, наследник BaseSettings, в котором определятся, что должно быть в .env файле. Здесь и везде далее используются type hints. Сам python не строго относится к их соблюдения, только подсказывает вам в IDE, что вы должны передать в качестве аргументов куда-либо, где указаны типы. Однако Pydantic проверяет соответствие указанных типов и полученных данных, что очень удобно и безопасно.
+https://fastapi.tiangolo.com/advanced/settings/#pydantic-settings
```
Comment (dyakovri) [inline docs/tvoy-ff/backend/config.md:4]: <>

**Pull request merged**